src/Motives/MotiveButtons.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export default function MotiveButtons({ handleMotiveClick }) {
+  return (
+    <ButtonList>
+      <button onClick={(event) => handleMotiveClick(event, 'Anerkennung')}>
+        Anerkennung
+      </button>
+      <button onClick={(event) => handleMotiveClick(event, 'Wichtigkeit')}>
+        Wichtigkeit
+      </button>
+      <button onClick={(event) => handleMotiveClick(event, 'Verlässlichkeit')}>
+        Verlässlichkeit
+      </button>
+      <button onClick={(event) => handleMotiveClick(event, 'Solidarität')}>
+        Solidarität
+      </button>
+      <button onClick={(event) => handleMotiveClick(event, 'Autonomie')}>
+        Autonomie
+      </button>
+      <button onClick={(event) => handleMotiveClick(event, 'Grenzen')}>
+        Grenzen
+      </button>
```

Comment:
Unsure of weather this is useful in your case, but you could save a few lines of code by creating the buttons iteratively (is that even a word??). 
```suggestion
      const values = ['Anerkennung, ...]
      values.map(text => <button onClick{event => handleMotiveClick(event, text)}>{text}</button>
```

Reply:
I changed it like you suggested. Makes the code a little shorter at least.